releasenotes/notes/csplayout-fb9deb7fc133acfd.yaml

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
I think this is from when `CSPLayout` was introduced. Can you update it to note that it is being included in levels 2 and 3 with default timeouts/call limits?

Reply:
Indeed. When the release was done this file was moved to 0.11.  Removing and adding new release notes.